content/posts/numpy/numpy2/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "18 years in the Making: NumPy 2.0 Released"
+date: 2024-06-16
+draft: false
+description: "
+18 years in the Making: NumPy 2.0 Released
```

Comment:
Would be good to have a more helpful summary here. When you paste a link to the blog post, most social media sites generate a card with the title + this description.

Reply:
Example text:

We announce the release of NumPy 2.0, which brings major improvements both
to the user API as well as the underlying implementation. We discuss the
motivation behind breaking changes, and how they impact users.

Reply:
Good idea. I tweaked this a bit and pushed something more descriptive.